modules/cluster-logging-log-store-status-viewing.adoc

```diff
@@ -101,31 +101,29 @@ status: <1>
   shardAllocationEnabled: all
 ----
 <1> In the output, the cluster status fields appear in the `status` stanza.
-<2> The status of the log store:
+<2> The status of the Elasticsearch log store:
 +
 * The number of active primary shards.
 * The number of active shards.
 * The number of shards that are initializing.
-* The number of log store data nodes.
-* The total number of log store nodes.
+* The number of Elasticsearch log store data nodes.
+* The total number of Elasticsearch log store nodes.
 * The number of pending tasks.
-* The log store status: `green`, `red`, `yellow`.
+* The Elasticsearch log store status: `green`, `red`, `yellow`.
 * The number of unassigned shards.
-<3> Any status conditions, if present. The log store status indicates the reasons from the scheduler if a pod could not be placed. Any events related to the following conditions are shown:
-* Container Waiting for both the log store and proxy containers.
-* Container Terminated for both the log store and proxy containers.
+<3> Any status conditions, if present. The Elasticsearch log store status indicates the reasons from the scheduler if a pod could not be placed. Any events related to the following conditions are shown:
+* Container Waiting for both the Elasticsearch log store and proxy containers.
+* Container Terminated for both the Elasticsearch log store and proxy containers.
 * Pod unschedulable.
 Also, a condition is shown for a number of issues; see *Example condition messages*.
-<4> The log store nodes in the cluster, with `upgradeStatus`.
-<5> The log store client, data, and master pods in the cluster, listed under 'failed`, `notReady`, or `ready` state.
+<4> The Elasticsearch log store nodes in the cluster, with `upgradeStatus`.
+<5> The Elasticsearch log store client, data, and master pods in the cluster, listed under 'failed`, `notReady`, or `ready` state.
 
 [id="cluster-logging-elasticsearch-status-message_{context}"]
```

Comment:
Same here as in the other one, out of scope but looks like a new module with ref content